<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="C_1">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D_1">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F_1">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="E_1">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="G_1">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_1">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="H_1">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>598</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>20</y>
      <width>71</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

	background: #ffffff; 
	border-style: outset;
	border-width: 2px;
	border-color: #000;</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_1d">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>20</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
text-color: #ffffff;
</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D_1d">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>20</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
text-color: #ffffff;
</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F_1d">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>20</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
text-color: #ffffff;
</string>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="G_1d">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>20</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
text-color: #ffffff;
</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_1d">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>22</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
text-color: #ffffff;
</string>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>20</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
text-color: #ffffff;
</string>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>574</x>
      <y>20</y>
      <width>51</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
text-color: #ffffff;
</string>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>621</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>D (Ре, первая октава) - S(Ы)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>E (Ми, первая октава) - D(В)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>G (Соль, первая октава) - G(П)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>D# (Ре диез, первая октава) - E(У)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>С (До, первая октава) - A(Ф)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>F (Фа, первая октава) - F(А)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>А (Ля, первая октава) - H(Р)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>H (Си, первая октава) - J(О)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>С# (До диез, первая октава) - W(Ц)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>F# (Фа диез, первая октава) - T(Е)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>G# (Соль диез, первая октава) - Y(Н)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>А# (Ля диез, первая октава) - U(Г)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
